src/resources/License/LicenseInterfaces.ts

```diff
@@ -97,6 +97,10 @@ export interface LicenseModel {
      * The type of the license
      */
     type: string;
+    /**
+     * The license of every machine learning model.
+     */
+    LicenseMachineLearningModel: LicenseMachineLearningModel;
```

Comment:
I tried it on swagger and the attribute is named `machineLearningModels`

```suggestion
    machineLearningModels: LicenseMachineLearningModel;
```

Reply:
Indeed thanks for the catch !